zeta/test-artifacts/init

```diff
@@ -0,0 +1,10 @@
+#!/bin/busybox sh
+
+echo "tcrypt: starting tcrypt test execution"
+
+./test-script.sh
+
+mkdir /proc
```

Comment:
how can we use `mkdir` directly, but not `mount`?

nit: you can probably avoid this by just creating `/proc` in initrd directory during build time

Reply:
It's a good question, can't tell for sure. It seems the busybox shell links some utilities out-of-the-box, but not others, but can't find exactly where that's happening.

Removed the /proc directory creation on the latest version, and we're now calling the busybox install command before invoking anything else so that we have all symlinks created.